dom.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dom.css" />
+    <title>JavaScript DOM Manipulation</title>
+    <style>
+      #experienceForm {
+        display: none;
+        flex-direction: column;
+        margin-top: 10px;
+      }
+      #experienceForm label,
+      #experienceForm textarea,
+      #experienceForm input,
+      #experienceForm button {
+        margin-bottom: 10px;
+        display: flex;
+        flex-direction: column;
+      }
```

Comment:
Please move these to the external stylesheet as well so that all the stylings are present in one file.

Reply:
fixed!